docs/glossary.md

```diff
@@ -0,0 +1,120 @@
+# Glossary
+
+## Accuracy
+The fraction of predictions a classification model has correctly identified. Formula:
+
+$$
+\text{accuracy} = \frac{\text{True Positives + True Negatives}}{\text{Total amount of data points}}
+$$
+
+## Application Programming Interface (API)
+An API allows independent applications to communicate with each other and exchange data.
+
+## Classification
+Classification refers to dividing a data set into multiple chunks, which are then considered "classes".
+
+## Confusion Matrix
+A confusion matrix is a table that is used to define the performance of a classification algorithm.
+It classifies the predictions to be either be true positive, true negative, false positive or false negative.
+
+## Decision Tree
+A Decision Tree represents the process of conditional evaluation in a tree diagram.
+
+## F1-Score
+The harmonic mean of [precision](#precision) and [recall](#recall). Formula:
+
+$$
+f_1 = 2 \cdot \frac{\text{precision} \cdot \text{recall}}{\text{precision} + \text{recall}}
+$$
+
+## False Negative (FN)
+An outcome is considered to be a false negative, if the data model has mistakenly predicted a value of negative class.
+
+## False Positive (FP)
+An outcome is considered to be a false positive, if the data model has mistakenly predicted a value of positive class.
+
+## Feature
+Each feature represents a measurable piece of data that can be used for analysis.
+It is analogous to a column within a table.
+
+## Linear Regression
+Linear Regression is the supervised Machine Learning model in which the model finds the best fit linear line between the independent and dependent variable
+i.e. it finds the linear relationship between the dependent and independent variable.
+
+Implemented in Safe-DS as [LinearRegression][safeds.ml.classical.regression.LinearRegression].
```

Comment:
Where applicable, please link to the documentation for the corresponding API element, as done here.